package_drupal_script.sh

```diff
@@ -102,27 +102,35 @@ for dockerurl in $DOCKER_URLS; do
 done
 
 # clone or refresh d8 clone
-if [ ! -d drupal8/.git ] ; then
-    git clone git://git.drupal.org/project/drupal.git drupal8
+mkdir -p sprint
+if [ ! -d sprint.tar.xz ] ; then
+    git clone git://git.drupal.org/project/drupal.git $STAGING_DIR/sprint/drupal8
 else
-    pushd $STAGING_DIR/drupal8
+    pushd $STAGING_DIR
+    tar xpvf sprint.tar.xz -C sprint
+    rm sprint.tar.xz
+    cd $STAGING_DIR/sprint/drupal8
     git pull
     popd
 fi
-pushd $STAGING_DIR/drupal8
+pushd $STAGING_DIR/sprint/drupal8
 composer install
-ddev config --docroot="" --sitename=drupal8 --apptype=drupal8
 
 # Copy licenses and COPYING notice.
 cp -r $REPO_DIR/licenses $STAGING_DIR/
 cp $REPO_DIR/COPYING $STAGING_DIR/
-
-# Grab a database for them to install to avoid the install process
-mkdir -p .db_dumps
-cp $REPO_DIR/databases/d8_installed_db.sql.gz .db_dumps
 popd
 
+cd $STAGING_DIR
+tar cfJ sprint.tar.xz -C sprint .
+rm -rf $STAGING_DIR/sprint
+
+if [ -f ${REPO_DIR}/package_additions.sh ]; then
+    # Package images for any additions.
+    source ${REPO_DIR}/package_additions.sh
+fi
+
 cd $STAGING_DIR_BASE
 tar -czf drupal_sprint_package.tar.gz $STAGING_DIR_NAME
 zip -r -q drupal_sprint_package.zip $STAGING_DIR_NAME
-printf "${GREEN}The sprint tarballs and zipballs are in $(ls $FINAL_TARGET_DIR/drupal_sprint_package*).${RESET}\n"
+printf "${GREEN}The sprint tarballs and zipballs are in $(ls $STAGING_DIR_BASE/drupal_sprint_package*).${RESET}\n"
```

Comment:
If things fail, let's say because a user was impatient and didn't wait for Docker to restart (*cough* me *cough*), then this message still prints out as a success even though it failed.

Reply:
Hmm, this is not end-user - this is in the packaging script. Were you thinking about commenting elsewhere?

Reply:
added a wait anyway.